app.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express")
+const {getTopics} = require("./controllers/topics.controllers")//controller
+
+const app = express();
+app.use(express.json());
```

Comment:
we don't need this line yet, take it out until the code actually requires you to put it in